tests/phpunit/tests/dependencies/scripts.php

```diff
@@ -1481,4 +1481,32 @@ public function data_wp_localize_script_data_formats() {
 			array( false, '[""]' ),
 		);
 	}
+
+	public function test_wp_external_wp_i18n_print_order() {
```

Comment:
```suggestion
	/*
	 * @ticket 55628
	 * @covers ::wp_set_script_translations
	 */
	public function test_wp_external_wp_i18n_print_order() {
```

Reply:
Make that `/**` no?

Reply:
> Make that `/**` no?

Yes, my mistake.

I'll push a fix to this branch so it can go in to trunk.